src/utilities.ts

```diff
@@ -15,6 +15,7 @@ const syntaxQuoteSymbol = '`';
 
 export function stripAnsi(str: string) {
     return str.replace(
+        // eslint-disable-next-line no-control-regex
```

Comment:
What does this mean?

Reply:
That disabled this eslint rule for the next line: https://eslint.org/docs/rules/no-control-regex#no-control-regex

Usually control characters in regex are a mistake but since we're stripping control characters here I'm disabling the rule just at this spot. this is the eslint escape hatch for when you really need it